github-workflow/aks-deploy.yaml

```diff
@@ -84,9 +84,8 @@ jobs:
       with:
         namespace: 'cluster-baseline-settings'
         manifests: |
-          cluster-manifests/cluster-baseline-settings/ns-cluster-baseline-settings.yaml
-          cluster-manifests/cluster-baseline-settings/flux.yaml
-
+          cluster-manifests/base/cluster-baseline-settings/ns.yaml
+          cluster-manifests/base/cluster-manifests/cluster-baseline-settings/flux.yaml
```

Comment:
In the next PR will you be applying the regional overlay then on these?

Reply:
regional overlay are all here now.

Done | Addressed from [2a12c](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/2a12cc40214b78ae9cbaa24412fa8a51cc9fcdc4) and [08e24](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/08e24fd5ac0fdf9ce0cc49e78b04d079b024405b)

Reply:
Looks nice, dude!